package.json

```diff
@@ -58,6 +58,7 @@
         "typescript": "^3.8.3"
     },
     "dependencies": {
-        "@react-native-firebase/dynamic-links": "^10.0.0"
+        "@react-native-firebase/dynamic-links": "^10.0.0",
+        "firebase-functions": "^3.14.0"
```

Comment:
@SukiCai do you happen to know why this is added and why we have a `package-lock.json`, the commit history shows that you created the file.

Reply:
@rlnsy, I have package-lock.json in every cloned local branch, seems like it is automatically generated for any operations where npm modifies either the node_modules tree or package.json. And for line63, I guess that one is merged from PR29?